include/graphblas/bsp1d/vector.hpp

```diff
@@ -2377,6 +2377,60 @@ namespace grb {
 #endif
 		}
 
+		/**
+		 * Creates a dense ALP/GraphBLAS vector. This constructor takes an initializer
+		 * list of values that will be copied into this vector.
+		 */
+		Vector( const std::initializer_list< D > &vals )
```

Comment:
This is by reference instead of by value. It should probably be by value?

Reply:
It's going to be copied either way in the vector so I guess it doesn't change anything